PERFORMANCE.md

```diff
@@ -0,0 +1,127 @@
+# Performance Results
+
+The performance of SmartSim is detailed below across various types of systems.
+
+Note that the first iteration can take longer (up to several seconds) than the rest of the execution. This
+is due to the DB loading libraries when the first RedisAI call is made. In the following plots, we excluded
+the first iteration time.
+
+## Inference Standard
+
+The following are scaling results from the cpp-inference scaling tests with ResNet-50
```

Comment:
It's worth providing an explanation of what these tests are. You don't have to go into as much detail as is in the paper, however the reader shouldn't have to go external to this document in order to understand what the test is and what the analogous use case is for their workflow